adm_program/installation/installation.php

```diff
@@ -685,6 +685,18 @@
                  , ('.$categoryIdSocialNetworks.', \'TEXT\', \'YAHOO_MESSENGER\',\'INS_YAHOO_MESSENGER\', NULL, \'yahoo.png\', NULL, 0, 8, '.$gCurrentUser->getValue('usr_id').',\''. DATETIME_NOW.'\') ';
     $db->query($sql);
 
+    // create user relation types
+    $sql = 'INSERT INTO '.TBL_USER_RELATION_TYPES.' (urt_id, urt_name, urt_name_male, urt_name_female, urt_id_inverse, urt_usr_id_create, urt_timestamp_create)
```

Comment:
You must not set the id. If the column is autoincrement then the database will set the id by itself.